aplus/urls.py

```diff
@@ -53,4 +53,9 @@
 if settings.DEBUG:
     import django.views.static
     urlpatterns.insert(0, url(r'^media/(?P<path>.*)$', django.views.static.serve,
-        { 'document_root': settings.MEDIA_ROOT }))
+                              {'document_root': settings.MEDIA_ROOT}))
+
+    import debug_toolbar
+    urlpatterns.insert(0,
+                       url(r'^__debug__/', include(debug_toolbar.urls)),
+                       )
```

Comment:
I think the following indentation looks nicer. We can also use the newer `path` function instead of the deprecated `url`. If regular expressions are really required, then the new function is `re_path`.
```suggestion
    urlpatterns.insert(
        0,
        path('__debug__/', include(debug_toolbar.urls)),
    )
```